src/model/base/FluxConditionPresentAssertion.js

```diff
@@ -150,6 +149,60 @@ class FluxConditionPresentAssertion extends FluxEntry {
         return undefined;
     }
 
+    // Returns the most recent toxicities within a specified time period
+    getMostRecentToxicities() {
+        // Set the max number of months prior to today that a toxicity can be
+        const numberOfMonths = 6;
+        const sinceDate = moment().subtract(numberOfMonths, 'months');
+        const sinceDateMoment = new moment(sinceDate, "D MMM YYYY");
+        const tox = this.getToxicities();
+        const sortedTox = tox.sort(this._toxicitiesTimeSorter);this.relatedEncounterReferencer
+        const mostRecentTox = {};
+        
+        sortedTox.forEach((t) => {
+            const id = t.type;
+
+            if (!id) return; 
```

Comment:
I think this should be a `continue` instead of a return; we don't want to exit the whole loop just because we ran into one toxicity without a type, right?

Reply:
Yeah that's what I initially thought too.  Apparently in a `forEach` you can't `continue` so you have to `return` to exit out of the current iteration.

https://stackoverflow.com/questions/31399411/go-to-next-iteration-in-javascript-foreach-loop/31399448

Reply:
lol that's wild.